cgt_calc/parsers/schwab_equity_award_json.py

```diff
@@ -216,9 +285,10 @@ def __init__(
                             )
 
                     quantity = (amount + fees) / price
+
         else:
             raise ParsingError(
-                file, f'Parsing for action {row["action"]} is not implemented!'
+                file, f"Parsing for action {row[names.action]} is not implemented!"
             )
 
         currency = "USD"
```

Comment:
Note: a few lines below (line 311+ in the new code/241+ in the old code - I can't add a comment there directly sadly) the `_normalize_split()` function seems to assume that the data is about Alphabet shares. I think some check on the stock symbol might be prudent to avoid accidentally scaling some other stock's values according to the Alphabet split, in cases where the user happens to also use Schwab as broker for employer-issued equity.

Reply:
The equity awards accounts and this JSON format are not unique to GOOG/GOOGL stocks, but I think this JSON format may be unique to equity awards accounts.

Reply:
It'd be great actually to have some JSON/CSV file in resources with information about splits to have general support for them instead of hardcoding a specific one directly in the code.